OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -51,7 +52,7 @@ public void Killed(Actor self, AttackInfo e)
 				return;
 
 			var warhead = e.Warhead as DamageWarhead;
-			if (info.DeathType != null && warhead != null && !warhead.DamageTypes.Overlaps(info.DeathType))
+			if (warhead != null && !warhead.DamageTypes.Overlaps(info.DeathType))
```

Comment:
This isn't right, `info.DeathType != null` needs a replacement (`.Count > 0`, I assume).